tailwind.config.js

```diff
@@ -460,6 +462,15 @@ module.exports = {
     //   black: '900',
     // },
     // gap: ({ theme }) => theme('spacing'),
+    gap: ({ theme }) => ({
```

Comment:
can we use `gap-y` `gap-x`?

Reply:
Yes, it generates them automatically with the same values. I added them to the story.